src/main/java/org/spongepowered/api/keyboard/KeyBinding.java

```diff
@@ -96,6 +98,34 @@ static Builder builder() {
         Builder displayName(Text displayName);
 
         /**
+         * Sets the press executor of this key binding, it will be called
+         * when a specific {@link Player} the key binding pressed.
+         *
+         * @param executor The press executor
+         * @return This builder for chaining
+         */
+        Builder pressExecutor(@Nullable BiConsumer<Player, KeyBinding> executor);
+
+        /**
+         * Sets the release executor of this key binding, it will be called
+         * when a specific {@link Player} the key binding release.
+         *
+         * @param executor The release executor
+         * @return This builder for chaining
+         */
+        Builder releaseExecutor(@Nullable BiConsumer<Player, KeyBinding> executor);
+
+        /**
+         * Sets the tick executor of this key binding, it will be called
+         * when a specific {@link Player} the key binding holds the key
+         * and a tick occurred.
+         *
+         * @param executor The tick executor
+         * @return This builder for chaining
+         */
+        Builder tickExecutor(@Nullable BiConsumer<Player, KeyBinding> executor);
```

Comment:
`reset` resets the entire builder, passing `null` to builder methods can clear that *single* builder value. Additionally, having to do the following will get plain annoying really fast:
```java
@Nullable BiConsumer<Player, KeyBinding> executor = null;
....
if (executor != null) {
    builder.tickExecutor(executor);
} else {
    builder.reset();
    // re-apply all other values
    builder.tickExecutor(executor);
}
```